src/mugs.js

```diff
@@ -468,6 +468,20 @@ define([
         }
     });
 
+    Object.defineProperty(Mug.prototype, "hashtagPath", {
```

Comment:
I did this because it was much easier changing the calls one by one instead of changing absolutePath everywhere all at once.

The only places using `absolutePath` are:
* ignoreButRetain: I decided it shouldn't  support hashtags since those are handwritten
* core.js: this is for drag/drop when the mug doesn't support rich text
* form.js: when adding hashtag translations
* atwho.js: to search when doing /data/
* saveToCase.js: doesn't support because key value widget
* modeliteration: uses attributes which isn't supported by js-xpath's hashtag implementation